tests/sample_qc/test_de_novo.py

```diff
@@ -13,8 +13,9 @@ class TestDeNovoMutation:
     """Test suite for de novo mutation functions."""
 
     @pytest.mark.parametrize(
-        "proband_pl, father_pl, mother_pl, diploid, hemi_x, hemi_y, freq_prior, min_pop_prior, expected_p_dn",
+        "proband_pl, father_pl, mother_pl, diploid, hemi_x, hemi_y, freq_prior, min_pop_prior, expected",
         [
+            # ✅ Valid test cases (should return numeric values)
```

Comment:
```suggestion
            # Valid test cases (should return expected numeric values)
```
I enjoy this emoji but don't think it's necessary here